docs/source/index.rst

```diff
@@ -22,29 +22,36 @@ would become hard to mantain very soon. You also might not want to tie your web
 app code to a specific vendor or firmware because that would make your software unflexible.
 
 If we had a single API we could let web developers focus on the task they need to accomplish
-rather than dealing with SNMP, SSH, different firmwares, different linux distributions and so on.
+rather than dealing with different firmwares, different linux distributions and so on.
 
 The goal of this project is to build that single API.
 
 ======================
 Status of this project
 ======================
 
-We are currently in 0.1 beta version.
+We are currently in 0.1 pre-alpha version.
 
-The 0.1 final version will be out by April 2015.
+The 0.1 final version will be out by August 2021.
+
+.. note::
+
+    The legacy versions of this project had support for SSH and HTTP for extracting information from
+    devices. To see how it worked, visit the
+    `0.1.0 pre-alpha release <https://github.com/openwisp/netengine/releases/tag/0.1.0a>`_ page on
```

Comment:
Read the docs provides versioned documentation. If it is not much work to set it up, then we can enable that and use that link here instead.

If above is not quickly doable, then we can link https://github.com/openwisp/netengine/tree/0.1.0a instead of the tag. We can update this link later while releasing the module

Reply:
Also, it was an alpha release, not pre-alpha.

Reply:
Add a note in https://github.com/openwisp/netengine/issues/56, if you want to delegate enabling versioned documentation. So we remember that we need to take care of this.